src/components/CreateThought/CreateThought.jsx

```diff
@@ -0,0 +1,82 @@
+import { useState, useEffect } from "react";
+import "./CreateThought.css"; //import styling
+
+export const CreateThought = () => {
+  const [newPost, setNewPost] = useState(""); // Initialize state for the new post input
+  const [errorMessage, setErrorMessage] = useState(""); // Initialize state for error messages
+
+  useEffect(() => {
+    // Check the length of `newPost` to provide feedback to the user with the rule of max.140 characters
+    if (newPost.length >= 141) {
+      setErrorMessage("Your message is too long 😔"); // Display an error message for long messages
+    } else {
+      // Clearing the error message if `newPost` is not too long
+      setErrorMessage("");
+    }
+  }, [newPost]); // The effect runs when `newPost` changes
+
+  const handleFormSubmit = async (event) => {
+    event.preventDefault(); // Prevent the default form submission behavior
+
+    if (newPost.length <= 4) {
+      setErrorMessage(
+        "Your message is too short, it needs at least 5 letters 😔"
+      ); // Display an error message for short messages
+    } else {
```

Comment:
This works well as far as I can tell, the error messages appears when you're trying to post a message that's not between 5-140 characters.